nginx_proxy/translations/en.yaml

```
@@ -24,6 +24,11 @@ configuration:
     name: Customize
     description: >-
       See the Documentation tab for more information about these options.
+  real_ip_from:
+    name: RealIpFrom
```

This can have spaces. And maybe give a hint what the consequences are in the name of the property already :thinking: 
```suggestion
    name: Real IP from (enables PROXY protocol)
```